examples/echo-bot-ts-cjs/package.json

```diff
@@ -17,6 +17,6 @@
     "@types/express": "^4.17.17",
     "@types/node": "^20.6.3",
     "rimraf": "^5.0.1",
-    "typescript": "^5.4.3"
+    "typescript": "^5.2.2"
```

Comment:
I manually confirmed that it can be compiled with five versions from 5.0 to 5.4.